programs/ziti-edge-tunnel/netif_driver/windows/tun.c

```diff
@@ -214,10 +222,6 @@ netif_driver tun_open(struct uv_loop_s *loop, uint32_t tun_ip, uint32_t dns_ip,
     uv_unref((uv_handle_t *) &tun->route_timer);
     uv_timer_start(&tun->route_timer, refresh_routes, ROUTE_REFRESH, ROUTE_REFRESH);
 
-    if (dns_ip) {
```

Comment:
`dns_ip` is now an unused argument here. it should be removed from the argilst for the windows implementation of `tun_open`